Zulip /README.md

```diff
@@ -0,0 +1,42 @@
+# Hi :smiley: :wave:
```

Comment:
the file should be located in the root folder, while we don't have anything else to put inside this folder. You can name it `zulip.md`